src/mail/view/MailFolderRow.ts

```diff
@@ -99,7 +100,7 @@ export class MailFolderRow implements Component<MailFolderRowAttrs> {
 						}, 5)
 					},
 				}),
-				rightButton && (this.hovered || editMode)
+				rightButton && (this.hovered || editMode) && !styles.isSingleColumnLayout()
```

Comment:
I would instead say  `styles.isUsingBottomNavigation()` or `isDesktopDevice()`, I don't think it makes sense on touchscreens at all